tests/test_enrichment.py

```diff
@@ -0,0 +1,342 @@
+            {
+                "bucket_name": "MIKE",
+                "county_lookup_column_1": "county_name",
+                "county_lookup_column_2": "region",
+                "county_lookup_column_3": "county",
+                "county_lookup_column_4": "marine",
+                "county_lookup_file": "countylookup",
+                "error_handler_arn": "Arrgh",
+                "identifier_column": "responder_id",
+                "marine_mismatch_check": "What",
+                "missing_county_check": "eh",
+                "missing_region_check": "oh",
+                "period_column": "period",
+                "responder_lookup_file": "responderlookup",
+            },
+        ):
+            client = boto3.client(
+                "s3",
+                region_name="eu-west-1",
+                aws_access_key_id="fake_access_key",
+                aws_secret_access_key="fake_secret_key",
+            )
+
+            client.create_bucket(Bucket="MIKE")
+            client.upload_file(
+                Filename="tests/fixtures/responder_county_lookup.json",
+                Bucket="MIKE",
+                Key="responderlookup",
+            )
+            client.upload_file(
+                Filename="tests/fixtures/county_marine_lookup.json",
+                Bucket="MIKE",
+                Key="countylookup",
+            )
+
+            with open("tests/fixtures/test_data.json", "r") as file:
+                testdata = file.read()
+
+            test_output = lambda_method_function.lambda_handler(testdata, "")
+            test_output = pd.read_json(test_output["data"])
+            print(test_output.columns.values)
+            assert "county" in test_output.columns.values
+            assert "county_name" in test_output.columns.values
+
+    @mock_sqs
+    def test_marshmallow_raises_method_exception(self):
+        sqs = boto3.resource("sqs", region_name="eu-west-2")
+        sqs.create_queue(QueueName="test_queue")
+        queue_url = sqs.get_queue_by_name(QueueName="test_queue").url
+        # Method
+        with mock.patch.dict(
+            lambda_method_function.os.environ, {"queue_url": queue_url}
+        ):
+            lambda_method_function.lambda_handler(
+                {"RuntimeVariables": {"checkpoint": 666}}, None
+            )
+            self.assertRaises(ValueError)
+
+    @mock_sqs
+    def test_marshmallow_raises_wrangler_exception(self):
+        sqs = boto3.resource("sqs", region_name="eu-west-2")
+        sqs.create_queue(QueueName="test_queue")
+        queue_url = sqs.get_queue_by_name(QueueName="test_queue").url
+        # Method
+        with mock.patch.dict(
+            lambda_wrangler_function.os.environ,
+            {"checkpoint": "1", "queue_url": queue_url},
+        ):
+            lambda_wrangler_function.lambda_handler(
+                {"RuntimeVariables": {"checkpoint": 666}}, None
+            )
+            self.assertRaises(ValueError)

```

Comment:
Should probably be a bit more specific here and check for what you're expecting to see failed....i.e the message